### PR TITLE
FIX: All payment types returned on GET/paymenttypes

### LINKED ISSUE #18: Bug: All payment types are being returned
When the client performs a GET request to the `/paymenttypes` resources, only the payment types added by the authenticated user should be returned.

Currently, all payment types in the system are being returned.

### LINKED ISSUE #19: Bug: Payment type info is incorrect
Customers are complaining that the expiration date on their payment types are incorrect.

### PR DESCRIPTION
**Pull request by DakotaLambert**
## Changes

- Profile view
> Added payment serializer to return the current users payment types (not all information)

- PaymentTypes 
> Changed LIST to get customer by auth.user, to return the payment types of the current user.

## Requests / Responses

**Request**

GET `/profile`

**Response**

```json
{
    "id": 5,
    "url": "http://localhost:8000/customers/5",
    "user": {
        "first_name": "Joe",
        "last_name": "Shepherd",
        "email": "joe@joeshepherd.com"
    },
    "phone_number": "555-1212",
    "address": "100 Endless Way",
    "payment_types": [
        {
            "merchant_name": "Visa",
            "customer": {
                "id": 5,
                "user": {
                    "first_name": "Joe",
                    "last_name": "Shepherd",
                    "email": "joe@joeshepherd.com"
                }
            },
            "expiration_date": "2020-01-01"
        }
    ],
    "recommends": []
}
```
**Request**

GET `/paymenttypes`

**Response**

```json
[
    {
        "id": 1,
        "url": "http://localhost:8000/paymenttypes/1",
        "merchant_name": "Visa",
        "account_number": "24ijio68948fj8439",
        "expiration_date": "2020-01-01",
        "create_date": "2019-11-11",
        "customer_id": 5
    }
]
```

## Testing

Description of how to test code...

- [ ] Run migrations
- [ ] Run test suite
- [ ] Seed database
- [ ] Open PostMan
- [ ] GET http://localhost:8000/paymenttypes
- [ ] Refer to /paymenttypes response
- [ ] GET http://localhost:8000/profile
- [ ] Refer to /profile response

## Related Issues

- Fixes #18 
- Fixes #19

### CODE REVIEW

File: bangazonapi/views/paymenttype.py [hunk 1]
```diff
@@ -20,34 +19,29 @@ class Meta:
...
-
         Returns:
             Response -- JSON serialized payment instance
         """
         new_payment = Payment()
         new_payment.merchant_name = request.data["merchant_name"]
         new_payment.account_number = request.data["account_number"]
-        new_payment.expiration_date = request.data["create_date"]
-        new_payment.create_date = request.data["expiration_date"]
+        new_payment.expiration_date = request.data["expiration_date"]
```
Comment (heymonicakay) [inline bangazonapi/views/paymenttype.py:34]: Fixes #19

**Review by heymonicakay — CHANGES_REQUESTED** [review]
Nice work! Issue #18 is linked in the PR description, but the code includes changes that fix issue #19 as also. Please revise your PR description to link all related issues. Ping me when that's done, and you'll be good to go!

File: bangazonapi/views/paymenttype.py (hunk 1 above)
Comment (DakotaLambert) [reply bangazonapi/views/paymenttype.py:34]: Edited!

**Review by heymonicakay — APPROVED** [review]

**Comment by heymonicakay** [comment]
@DakotaLambert Thanks! Approved!

**Pull request merged**